barretenberg/cpp/src/barretenberg/vm/avm_trace/avm_trace.hpp

```diff
@@ -21,6 +21,11 @@
 namespace bb::avm_trace {
 
 using Row = bb::AvmFullRow<bb::fr>;
+enum class AddressingMode {
+    DIRECT,
+    INDIRECT,
+};
+using ResolvedAddr = std::pair<AddressingMode, uint32_t>;
```

Comment:
Do use a struct please! :)

Also, IIUC, this is not a resolved address. It's the original address, with the addressing mode.

Reply:
I agree.